main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```diff
@@ -560,6 +560,54 @@ object Monomorpher {
           }
       }.get // This is safe since the last case can always match.
 
+    case LoweredAst.Expr.JvmReflection(exp, true, tpe, eff, loc) =>
```

Comment:
Does pattern matching allow doing `proxy = true` here?

Reply:
sadly no :L I could add a enum for it.